src/Contracts/ConnectionInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Yiisoft\Db\Contracts;
+
+interface ConnectionInterface
```

Comment:
1. I don't think it's a good idea to move interfaces into `Contracts`. It's better to place them according to their use, not according to their type.
2. There's no sense in adding `ConnectionInterface`. There's only a single implementation and no other implementations are planned.